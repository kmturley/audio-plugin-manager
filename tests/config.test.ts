import { cli, CliOutput, getLastLine } from './shared';
import path from 'path';

const PLUGIN_DIR: string = path.join('test', 'plugins');

test('Config set', async () => {
  const output: CliOutput = await cli(`config set pluginFolder "${PLUGIN_DIR}"`);
  expect(getLastLine(output)).toEqual(PLUGIN_DIR);
});

test('Config get', async () => {
  const output: CliOutput = await cli(`config get pluginFolder`);
  expect(getLastLine(output)).toEqual(PLUGIN_DIR);
});
