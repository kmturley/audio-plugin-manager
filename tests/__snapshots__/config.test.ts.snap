// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Config get 1`] = `
"test/plugins
"
`;

exports[`Config set 1`] = `
"test/plugins
"
`;
